webui/react/src/pages/Cluster/ClusterOverview.tsx

```diff
@@ -171,17 +180,30 @@ const ClusterOverview: React.FC = () => {
               {overview.CPU.total - overview.CPU.available} <small>/ {overview.CPU.total}</small>
             </OverviewStats> : null
           }
-          <OverviewStats title="CPU Containers Running">
-            {auxContainers.running} <small>/ {auxContainers.total}</small>
-          </OverviewStats>
+          {auxContainers.total ?
+            <OverviewStats title="Aux Containers Running">
+              {auxContainers.running} <small>/ {auxContainers.total}</small>
+            </OverviewStats>: null
+          }
         </Grid>
       </Section>
       <Section hideTitle title="Overall Allocation">
-        <SlotAllocationBar
-          resourceStates={gpuSlotStates}
-          showLegends
-          size={ShirtSize.enormous}
-          totalSlots={overview.GPU.total} />
+        {overview.GPU.total > 0 &&
+          <SlotAllocationBar
+            resourceStates={gpuSlotStates}
+            showLegends
+            size={ShirtSize.enormous}
+            title={`Compute (${ResourceType.GPU})`}
+            totalSlots={overview.GPU.total} />
+        }
+        {overview.CPU.total > 0 &&
+          <SlotAllocationBar
+            resourceStates={cpuSlotStates}
+            showLegends
+            size={ShirtSize.enormous}
+            title={ResourceType.CPU}
```

Comment:
should that be the similar to line 196, `Compute (${ResourceType.CPU})`?

Reply:
aah yeah good catch. I'm working on a fixture (stubbed API response) to showcase these changes as we talked about.